flang/lib/Lower/ConvertExpr.cpp

```diff
@@ -331,9 +331,15 @@ class ExprLowering {
   genval(const Fortran::evaluate::ProcedureDesignator &proc) {
     if (const auto *intrinsic = proc.GetSpecificIntrinsic()) {
       auto signature = Fortran::lower::translateSignature(proc, converter);
+      // Intrinsic lowering is based on the generic name, so retrieve it here in
+      // case it is different from the specific name. The type of the specific
```

Comment:
"the type of the specific" should have a noun after it. Intrinsic?

Reply:
Fixed, thanks